src/parse.rs

```diff
@@ -18,6 +18,10 @@ use self::lrtable::{Minimiser, from_yacc};
 use self::cfgrammar::TIdx;
 use self::cfgrammar::yacc::{yacc_grm, YaccGrammar, YaccKind};
 
+// This can be arbitrary, ultimately it doesn't matter what the placeholder's
+// value is, because it is switched out almost immediately.
+const PLACEHOLDER: usize = 666;
```

Comment:
I'd suggest making it `usize::max_val()`, because that's unlikely to be confused for a real jump position.

Reply:
fixed